README.md

```diff
@@ -30,6 +30,15 @@
     * Deploy and maintain your ML models in production reliably and efficiently.
     * Weaviate's modular design allows you to easily package any custom trained model you want.
     * Smooth and accelerated handover of your Machine Learning models to engineers.
+## Install
```

Comment:
Install section is an essential part of every readme; some examples: 
https://github.com/apache/airflow#installing-from-pypi
https://github.com/dbt-labs/dbt-core#getting-started
https://github.com/prestodb/presto#running-the-cli